Assignment1/Assignment1/CalculateSimpleInterest.m

```diff
@@ -0,0 +1,21 @@
+
+
+#import "CalculateSimpleInterest.h"
+
+@implementation CalculateSimpleInterest
+
+
+
+@synthesize rate;
+@synthesize timePeriod;
+
+float const simpleInterestDivisor=100;
+
++(float)simpleInterestCalculatorWithAmount:(float)principalAmount withinterest:(float)rate withtime:(float)timePeriod  {
```

Comment:
If you have declared instance method in .h file, you have to care about its definition part also :
 see difference 
-(float)simpleInterestCalculatorWithAmount :(float)principalAmount withinterest:(float) rate withtime : (float)timePeriod; in .h file
+(float)simpleInterestCalculatorWithAmount:(float)principalAmount withinterest:(float)rate withtime:(float)timePeriod in .m file